src/sdl2/rect.rs

```diff
@@ -299,7 +299,35 @@ impl Rect {
         self.raw.h = clamp_size(height) as i32;
     }
 
-    /// Checks whether this rect contains a given point.
+    /// Checks whether this rect contains a given point, or touches it on the
+    /// right and/or bottom edges.  This method is deprecated in favor of
+    /// [`Rect::contains_point`](#method.contains_point).
+    ///
+    /// For historical reasons, this method differs in behavior from
+    /// [`SDL_PointInRect`](https://wiki.libsdl.org/SDL_PointInRect) by
+    /// including points along the bottom and right edges of the rectangle, so
+    /// that a 1-by-1 rectangle actually covers an area of four points, not
+    /// one.
```

Comment:
Can you please link to issue #569 here ? The notice here is great but it would be even better you there was a link to the original issue / discussion so curious users can see what it is about exactly 😃